3-semantic-html/index.html

```diff
@@ -0,0 +1,30 @@
+
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
```suggestion
<html lang="es">
```
El contenido está en español